src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -404,13 +404,13 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
     this.setState({ item: item, category: item.data.category, rarity: item.rarity })
   }
 
-  handleCategoryChange = (_event: React.SyntheticEvent<HTMLElement, Event>, { value }: DropdownProps) => {
+  handleCategoryChange = async (_event: React.SyntheticEvent<HTMLElement, Event>, { value }: DropdownProps) => {
```

Comment:
This function didn't have an async keyword on it. What was the reason behind its change?
There's also the possibility of removing the `async` keyword from it and returning `this.updateThumbnailByCategory(category)`

Reply:
the reason to add the async keyword was that the method `this.updateThumbnailByCategory`is async and broke an eslint rule type but returning it works without problems.